### PR TITLE
[16.0][MIG] product_fao_fishing: Migration to version 16.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT43023

@ernestotejeda please review!

### CODE REVIEW

File: product_fao_fishing/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Product FAO Fishing",
+    "summary": "Set fishing areas and capture technology",
+    "version": "16.0.1.0.1",
```
Comment (CarlosRoca13) [inline product_fao_fishing/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

File: product_fao_fishing/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                [
+                    ("product_tmpl_id", "=", template.id),
+                    ("attribute_id", "=", fao_fishing_area_attribute.id),
+                ]
+            )
+            template.fao_fishing_area_ids = attribute_line.value_ids
+
+    def _compute_fao_fishing_technique_ids(self):
+        """
+        Helper method to retrieve the fishing areas from product attributes
```
Comment (CarlosRoca13) [inline product_fao_fishing/models/product_template.py]:
```suggestion
        Helper method to retrieve the fishing techniques from product attributes
```

**Review by CarlosRoca13 — COMMENTED** [review]
Two minnor comments

**Review by CarlosRoca13 — COMMENTED** [review]
Two minnor comments

File: product_fao_fishing/models/product_template.py (hunk 2 above)
Comment (stefan-tecnativa) [reply product_fao_fishing/models/product_template.py]: Changes done, thanks for the comments.

**Review by CarlosRoca13 — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**